include/rcutils/strerror.h

```diff
@@ -0,0 +1,45 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef RCUTILS__STRERROR_H_
+#define RCUTILS__STRERROR_H_
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+#include "rcutils/visibility_control.h"
+
+/// Retrieve the string corresponding to the last system error.
+/* This function retrieves the value of errno, and calls the system-specific
```

Comment:
nitpick: this isn't keeping with the style in `rcutils`, e.g. https://github.com/ros2/rcutils/blob/32486f43f36e9fb09e103c2d10345d876161ea17/include/rcutils/allocator.h#L31 or https://github.com/ros2/rcutils/blob/32486f43f36e9fb09e103c2d10345d876161ea17/include/rcutils/find.h#L27, just to randomly select a few.

Reply:
Arg.  The style is inconsistent, and I always happen to pick the examples that are "different".  I can change it to the `**`.